tsconfig.json

```
@@ -14,5 +14,6 @@
       "@apis/*": ["src/apis/*"]
       // "@types/*": ["src/types/*"] // @types is typescript library, so use @/types/* instead of @types/*
     }
-  }
+  },
+  "exclude": ["node_modules", "node_modules/react-native-naver-map"]
```

```suggestion
  "exclude": ["node_modules"]
```

node_modules 하위 디렉토리는 없어도 괜찮지 않나요?

---

맞습니다 ㅋㅋ 하도 tsc 걸려서 추가했었는데 결국 react-native-naver-map.d.ts 추가로 해결.. 하위디렉토리는 지울게요~